bayesian/generate_manifest.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+from lxml.etree import SubElement, Element, ElementTree
+from io import BytesIO
+
+
+class PomXMLTemplate:
+
+    def __init__(self, json_data=None):
+        if json_data:
+            self._data = dict(json_data)
+        else:
+            raise ValueError
```

Comment:
A descriptive error should be present - pass what is wrong to exception constructor.